site/content/docs/main/plugin-release-instructions.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Releasing Velero plugins
+layout: docs
+toc: "true"
+---
+
+Velero plugins maintained by the core maintainers do not have any shipped binaries, only container images, so there is no need to invoke a GoReleaser script.
+Container images are built via a CI job on git push.
+
+Plugins the Velero core team is responsible include all those listed in [the Velero-supported providers list]({{< ref "supported-providers.md#velero-supported-providers" >}}) _except_ the vSphere plugin.
```

Comment:
Trying this shortcode to link directly to the anchor, but it wasn't working locally despite a lot of different permutations. I'm seeing if it's solely something in my own set up, or something truly wrong.

Reply:
Yeah, the links are still broken for some reason. It looks like the examples in https://gohugo.io/content-management/shortcodes/#ref-and-relref, but I'm not exactly sure what's going on.

Reply:
@carlisia @zubron My desire here was to link directly to the table on the page with an anchor. As the primary consumers of the document, do you think it's worth spending more time trying to figure out why the shortcode isn't generating the right URL, or do you think it's acceptable to just link to the page?

Another alternative is doing a direct URL link, I suppose.

Reply:
I can't get a direct link to work, either, so I'm removing it for the sake of moving the PR forward. There might be some Hugo setting I'm missing that causes the actual URL to end up empty.